### ISSUE TITLE
Layout Bug FieldRowPanel with heading

### ISSUE THREAD

**Issue by onno-timmerman** [wagtail/wagtail#5691]
If you make the combination with FieldRowPanel an heading the header overlaps the fields.

```
  FieldRowPanel(
            [
                FieldPanel('year'),
                FieldPanel('call')
            ], heading="year and call"
        ),
```

**Comment by loicteixeira** [comment]
Thanks for the report @onno-timmerman. However, when you created the issue, you should have been presented with a base template. Could you update your entry with the missing details?

**Comment by onno-timmerman** [comment]
Found a bug? Please fill out the sections below. 👍

### Issue Summary
When you make a fieldrowpanel in combination with a heading the heading will overlap the fields.


### Steps to Reproduce
Create a model with the following field panels
```
  FieldRowPanel(
            [
                FieldPanel('some charfield'),
                FieldPanel('some other charfield')
            ], heading="any header will do"
        ),
```
See the result in the admin. You will notice that the header is overlapping the fields. 
I would expect the header to be nice above to fieldrow.

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: (yes)
Tested in wagtail 3.7 


### Technical details

* Python version: Run `python 3.7`.
* Django version: 2.2.
* Wagtail version: 3.7.
* Browser version: Chrome 78.

**Comment by lb-** [comment]
Resolved via #5707 / 5a8f468c2d62964e64c07448377b720f60392a5c
